docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -226,9 +215,9 @@ Contributor Workflow
 Changelogs
 ----------
 
-Collections are required to include a changelog. To give a consistent feel for changelogs across collections and ensure changelogs exist for collections included in the ``ansible`` package we suggest you use `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_ to maintain and generate this but other options exist. Preferred (in descending order):
+Collections MUST include a changelog. You can choose one of the following options:
 
-#. Use antsibull-changelog (preferred).
+#. Use `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_: to give a consistent feel for changelogs across collections included in the ``ansible`` package. This is the **recommended** tool to maintain and generate your changelog.
 #. Provide ``changelogs/changelog.yaml`` in the `correct format <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md>`_. (You can use ``antsibull-lint changelog-yaml /path/to/changelog.yaml`` to validate the format.)
 #. Provide a link to the changelog file (self-hosted) (not recommended).
```

Comment:
Should we still allow collections to use something else than `changelogs/changelog.yaml`? I really like the unified changelog.

If someone doesn't want to create changelog fragments, they can always use [antsichaut](https://github.com/ansible-community/antsichaut/) or create something similar for their use-case.

Reply:
The unified changelog SGTM, please suggest wording

Reply:
The current formulation sounds good to me.